ee-9/feature-pack/pom.xml

```diff
@@ -534,6 +535,10 @@
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-client\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-extension\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-web-infinispan\z</exclude>
+                                <!-- Don't transform Messaging modules -->
+                                <exclude>org.hornetq:.\z</exclude>
+                                <exclude>org.wildfly:wildfly-messaging-activemq-injection\z</exclude>
+                                <exclude>org.wildfly:wildfly-messaging-activemq-subsystem\z</exclude>
```

Comment:
This should be wildfly-messaging-activemq-subsystem-jakarta with a comment

'Uses EE 8 JSON-P internally' 

(that's what the verbose logging says is the reason it transforms the -jakarta artifact

rg.wildfly:wildfly-messaging-activemq-subsystem is not needed; see my comment at L757